Production/crab/config/2018/EGamma.txt

```diff
@@ -0,0 +1,7 @@
+lumiMask=/afs/cern.ch/cms/CAF/certification/Collisions18/13TeV/ReReco/Cert_314472-325175_13TeV_17SeptEarlyReReco2018ABC_PromptEraD_Collisions18_JSON.txt
```

Comment:
- lumi file is wrong and for other 2018 data
- saveGenBosonInfo and saveGenJetInfo should be False for the data